azerothcore/azerothcore-wotlk
# fix(CORE/Raid): Lich King “Pain and Suffering” snapping and “Shadow Trap” not targeting tanks

## CHANGES PROPOSED:
- Updated boss facing angle to face the last target of Pain And Suffering
- Removed tank requirement for target of Shadow Trap


## ISSUES ADDRESSED:
- closes https://github.com/azerothcore/azerothcore-wotlk/issues/2826



## TESTS PERFORMED:
- Created new character
- Cleared all icc bosses 25HC to get to LK
- Started LK fight
- Waited a bit for LK to cast Shadow Trap and confirm placement
- Move around the platform and confirm that LK is following you with every strike of pain and suffering



## HOW TO TEST THE CHANGES:

1. Created new character
2. Cleared all ICC bosses 25HC
3. Started LK RP
4. Waited a bit for LK to cast Shadow Trap and confirm placement
5. Move out of shadow trap so you don't get kicked out of the platform.
5. Deal damage to change phase
6. Move around the platform and confirm that LK is following you with every strike of pain and suffering


## KNOWN ISSUES AND TODO LIST:


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
